P1.txt

```diff
@@ -0,0 +1,12 @@
+val a: Integer
+read(a)

```

Comment:
The program cannot contain i/o statements as the ioStatement nonterminal cannot be obtained from the starting nonterminal (program)